product-service/docs/swagger.yml

```diff
@@ -0,0 +1,106 @@
+openapi: 3.0.3
+info:
```

Comment:
do you write this file manually?
You can try using serverless-auto-swagger plugin